File: app/src/main/java/cn/alvkeke/dropto/ui/adapter/MgmtViewPagerAdapter.java
package cn.alvkeke.dropto.ui.adapter;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentActivity;
import androidx.viewpager2.adapter.FragmentStateAdapter;

import cn.alvkeke.dropto.ui.fragment.MgmtStorageFragment;

public class MgmtViewPagerAdapter extends FragmentStateAdapter {

    public MgmtViewPagerAdapter(@NonNull FragmentActivity fragmentActivity) {
        super(fragmentActivity);
    }

    @NonNull
    @Override
    public Fragment createFragment(int i) {
        return new MgmtStorageFragment();
    }

    @Override
    public int getItemCount() {
        return 1;
    }

}

File: app/src/main/java/cn/alvkeke/dropto/ui/adapter/MgmtViewPagerAdapter.kt
package cn.alvkeke.dropto.ui.adapter

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentActivity
import androidx.viewpager2.adapter.FragmentStateAdapter
import cn.alvkeke.dropto.ui.fragment.MgmtStorageFragment

class MgmtViewPagerAdapter(fragmentActivity: FragmentActivity) :
    FragmentStateAdapter(fragmentActivity) {
    override fun createFragment(i: Int): Fragment {
        return MgmtStorageFragment()
    }

    override fun getItemCount(): Int {
        return 1
    }
}
